wandb/proto/wandb_settings.proto

```diff
@@ -93,6 +93,9 @@ message Settings {
   // The W&B entity, like a user or a team.
   google.protobuf.StringValue entity = 69;
 
+  // The W&B organization.
```

Comment:
you should probably also add it in the wandb_settings.py

Reply:
sorry i missed this from last commit. Just finished adding it in wandb_settings.py as well